### DISCUSSION TITLE
How to calculate the distance from a point to a line？

### DISCUSSION THREAD

**Discussion by Lydanne** [georust/geo#878] [Q&A] [answered]
How to calculate the distance from a point to a line?

I wrote this and he reported an error：

![image](https://user-images.githubusercontent.com/39021696/179217846-1fdb3a2a-0feb-4c34-b899-cdd4264a810c.png)

**Comment by michaelkirk** [comment, marked as answer]
What is the error message that you're seeing?

**Reply by Lydanne** [reply]
![image](https://user-images.githubusercontent.com/39021696/179231873-be475253-9ba8-4369-9941-0e0b10896a05.png)

**Reply by Lydanne** [reply]
@michaelkirk This way

**Reply by Lydanne** [reply]
I solved it, all values ​​must be of float type
